lib/perl/Genome/InstrumentData/AlignmentResult.pm

```diff
@@ -514,19 +521,26 @@ sub create {
 
     # STEP 14: RESIZE THE DISK
     # TODO: move this into the actual original allocation so we don't need to do this
```

Comment:
Do you still agree with this TODO?  Is it worth doing, or can the TODO be removed?